packages/itwin/tree-widget/package.json

```diff
@@ -52,15 +52,15 @@
     "@itwin/core-frontend": "^4.0.0",
     "@itwin/core-react": "^4.10.0",
     "@itwin/ecschema-metadata": "^4.0.0",
-    "@itwin/presentation-components": "^5.0.0",
+    "@itwin/presentation-components": "^5.3.2",
```

Comment:
Why do we need to bump `presentation-components` peer dep?

Reply:
I think it should be enough to bump dev dep. No need to bump peer.

Reply:
Changed.